sevntu-checks/config/ant-phase-verify.xml

```diff
@@ -42,6 +42,73 @@
     </tstamp>
     <echo level="info">Checkstyle finished (${checkstyle.configLocation}) : ${FINISHED}</echo>
 
+    <tstamp>
+      <format property="STARTED" pattern="dd/MM/yyyy hh:mm:ss aa"/>
+    </tstamp>
+    <echo level="info">Checkstyle started (${checkstyle.nonMain.configLocation}): ${STARTED}</echo>
+
+    <checkstyle config="${checkstyle.nonMain.configLocation}"
+                failOnViolation="true"
+                failureProperty="checkstyle.failure.property"
+                executeIgnoredModules="true"
+                >
+      <path>
+        <fileset dir="${basedir}/.." includes="**/*">
```

Comment:
This is one of the main differences from main repo. This run is in a sub-directory, so we have to go up one to collect the entire repo. This also means a lot of the excludes have to be adjusted and that is why they start with "**/".